src/GitHub.Exports/Services/GitService.cs

```diff
@@ -70,5 +72,36 @@ public UriString GetRemoteUri(IRepository repo, string remote = "origin")
         }
 
         public static IGitService GitServiceHelper => VisualStudio.Services.DefaultExportProvider.GetExportedValueOrDefault<IGitService>() ?? new GitService();
+
+        /// <summary>
+        /// Finds the latest pushed commit of a file and returns the sha of that commit. Returns null when no commits have 
+        /// been found in any remote branches or the current local branch. 
+        /// </summary>
+        /// <param name="path">The local path of the file. This cannot be null.</param>
```

Comment:
I think `The local path of a repository or a file inside a repository. This cannot be null.` will be clearer.